common/src/main/java/com/wynntils/core/features/overlays/OverlayManager.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.core.features.overlays;
+
+import com.wynntils.core.Reference;
+import com.wynntils.core.WynntilsMod;
+import com.wynntils.core.features.Feature;
+import com.wynntils.core.features.overlays.annotations.Overlay;
+import com.wynntils.mc.event.RenderEvent;
+import com.wynntils.utils.objects.Pair;
+import java.lang.reflect.InvocationTargetException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+public class OverlayManager {
+    private static final Map<OverlayBase, OverlayPosition> overlayInstanceMap = new HashMap<>();
+
+    private static final Map<Class<? extends Feature>, List<Pair<OverlayBase, OverlayPosition>>> overlaysFeatureMap =
```

Comment:
If you heed my advice and make the OverlayManager only handle instances of Overlays, maybe this point will be moot, but there is some kind of redundancy here. 

If you have a separate mapping between an OverlayBase and a position, you don't also need to include the position in the mapping from feature to list of overlays. So the overlaysInstanceMap is really a "overlayPositionMap", mapping from overlay instance to position (which is the "configy" stuff about the placement), and "overlaysFeatureMap" (maybe skip the plural s?) is a map from Feature (class, as chosen here, or possibly instance instead?) to  a List of OverlayBase instances that the Feature holds.